src/lib/common/include/sol-pin-mux-modules.h

```diff
@@ -32,6 +32,11 @@
 
```

Comment:
Do not increase API version until we release a new version. During development numbered should be the same